shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/converters/ManagedBridgeConverter.java

```diff
@@ -0,0 +1,51 @@
+package com.redhat.service.smartevents.shard.operator.v2.converters;
+
+import com.redhat.service.smartevents.infra.core.api.dto.KafkaConnectionDTO;
+import com.redhat.service.smartevents.infra.core.exceptions.definitions.platform.InvalidURLException;
+import com.redhat.service.smartevents.infra.v2.api.models.dto.BridgeDTO;
+import com.redhat.service.smartevents.shard.operator.v2.resources.*;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+@ApplicationScoped
```

Comment:
@vaibhavjainwiz We don't need this annotation if the method is static as there is no need to `@Inject` the bean any more. So it comes down to your preference on style on what you want to do here. If you prefer the `static` method approach, then simply drop the `@ApplicationScoped`. If you prefer having the bean `@Inject`, then remove the `static` definition on the method.

cheers,

Rob

Reply:
@vaibhavjainwiz Unit test as well please :-)

Reply:
by mistake.. fix it now